### DISCUSSION TITLE
Confused about Trip Distance

### DISCUSSION THREAD

**Discussion by iandavo2** [agronick/aa-torque#80] [Q&A]
Thanks agronick for getting this up & running! Awesome work. 
I can't figure out where the value for Trip Distance comes from. In AA Torque it shows some value that comes from I know not where, increments it as I drive and the value is sticky, ie doesn't reset when the engine starts. Torque Pro on the other hand resets the Trip Distance counter every time the engine starts, which makes it effectively useless, or when the "Trip Reset" display button is pressed. AA Torque doesn't reset when a Torque Pro button for "Trip Reset" is pressed, nor when any of the vehicle odometers are reset. I'd like to use this if I could only figure it out... if anyone could point me in the right direction I'd appreciate it!
Cheers
Ian

**Comment by agronick** [comment]
I just ask Torque for data [here](https://github.com/agronick/aa-torque/blob/master/app/src/main/java/com/aatorque/stats/TorqueRefresher.kt#L62). It doesn't know what PID it is requesting it for. It is just sending a PID and getting a number back. If something is not coming back correctly it is because Torque isn't exporting it through the service correctly.
